### ISSUE TITLE
No error for missing escaping anymore in some cases

### ISSUE THREAD

**Issue by kkmuffme** [WordPress/WordPress-Coding-Standards#1977]
## Bug Description
`_e( 'Login', 'example' );`

Until recently this returned an error for 
>All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks)

For some reason it doesn't anymore. (specifically this function, when we do e.g. `echo $_GET['hello'];` we still get the error correctly)

## Minimal Code Snippet

```php
_e( 'Login', 'example' );
```

## Error Code
WordPress.Security.EscapeOutput.OutputNotEscaped

## Environment

| Question               | Answer
| ------------------------| -------
| PHP version             | 7.4.14
| PHP_CodeSniffer version | 3.5.8
| WPCS version            | 2.3.0
| WPCS install type       | Composer global

**Comment by dingo-d** [comment]
Works well for me (that is I get reported error). Can you provide your ruleset?

**Comment by kkmuffme** [comment]
Ah sorry, this seems to happen when using this ruleset in addition to 
`<rule ref="WordPress-VIP-Go"/>`

Seems like WP VIP Go disables this check for _e within this ruleset.
Should I open an issue there?

**Comment by GaryJones** [comment]
@kkmuffme No thanks, that behaviour is intentional. Language files are handled in a different way there, so it's far less open to a bad string being present. Translations are trusted.

https://github.com/Automattic/VIP-Coding-Standards/blob/fe2c16e1b9f3bb243c5880123ae02d0e08c547a8/WordPress-VIP-Go/ruleset.xml#L209-L211
